src/webportal/src/app/job/job-view/fabric/JobList/Table.jsx

```diff
@@ -169,22 +168,25 @@ export default function Table() {
         Failed: MessageBarType.remove,
         Stopped: MessageBarType.blocked,
       }[statusText];
-      const rootStyle = {
+      
+      const rootStyle = {backgroundColor: 'transparent'};
+      /** @type {import('@uifabric/styling').IStyle} */
+      const iconContainerStyle = {marginTop: 8, marginBottom: 8, marginLeft: 8};
+      /** @type {import('@uifabric/styling').IStyle} */
+      const iconStyle = {
+        color: 'white', borderRadius: '50%', 
         backgroundColor: {
-          Waiting: '#FCD116',
-          Running: '#0071BC',
-          Stopping: '#0071BC',
-          Succeeded: '#7FBA00',
-          Failed: '#E81123',
+          Waiting: '#F9B61A',
```

Comment:
status badge component has been extracted to src/app/components/status-badge.jsx.
Please require and tweak the common component.